src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -350,6 +361,9 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
             zoomControl.addTo(map);
             initializeOverlays();
             initializeLayerControl.apply(this, null);
+            if (currentLocationMarker) {
+                currentLocationMarker.addTo(map);
+            }
```

Comment:
This doesn't work here, because `currentLocationMarker` gets initialized in the `on('locationfound', ...`, which might (in my experience so far, usually does) fire after this runs, so the marker is still null when this checks it and the layer never shows up.  I think it would make sense to remove this from here and put the `currentLocationMarker.addTo(map);` in the `if (!currentLocationMarker)` block of `showCurrentLocation`.

Actually, it looks like my proposed solution causes the marker to show behind the form on the home screen.  
![image](https://user-images.githubusercontent.com/6598836/28881421-102441ae-7776-11e7-8049-0334b34469b9.png)

Or that could be an independent problem that was masked by the failure of the marker to show up at all.